build/parseScript.c

```diff
@@ -451,6 +465,12 @@ int parseScript(rpmSpec spec, int parsePart)
 	      case PART_POSTTRANS:
 		pkg->postTransFile = xstrdup(file);
 		break;
+	      case PART_PREUNTRANS:
+		pkg->preTransFile = xstrdup(file);
```

Comment:
Shouldn't this rather be `preunTransFile`?

Reply:
Oops. Yep, good spotting!